src/main.cpp

```diff
@@ -7,6 +7,8 @@ int main(int argc, char *argv[])
     QCoreApplication::setOrganizationName("mkeeter");
     QCoreApplication::setOrganizationDomain("https://github.com/mkeeter/fstl");
     QCoreApplication::setApplicationName("fstl");
+    QCoreApplication::setApplicationVersion(FSTL_VERSION);
+    printf("Version: %s\n", FSTL_VERSION);
```

Comment:
Since the version is now visible in the app I would remove the `printf` here.

Reply:
Sounds good; Done.